### PR TITLE
Makes it optional to install amzn rpms from AMI Utils project

### PR DESCRIPTION
**Pull request by lorengordon**

### CODE REVIEW

**Comment by lorengordon** [comment]
go codebuild go

**Comment by lorengordon** [comment]
Depends on: https://github.com/plus3it/amigen7/pull/117

**Comment by lorengordon** [comment]
go codebuild go

**Comment by lorengordon** [comment]
go codebuild go

**Review by ferricoxide — COMMENTED** [review]
With respect to the `max_retries`: is this likely to ever be required for other builder-defs (esp. Azure – maybe have @nortiz34 test)?

File: spel/scripts/amigen7-build.sh [hunk 1]
```diff
@@ -18,7 +18,7 @@ AMIGENROOTNM="${SPEL_AMIGENROOTNM:-UNDEF}"
 AMIGENSOURCE="${SPEL_AMIGENSOURCE:-https://github.com/plus3it/AMIgen7.git}"
 AMIGENSTORLAY="${SPEL_AMIGENSTORLAY:-/:rootVol:4,swap:swapVol:2,/home:homeVol:1,/var:varVol:2,/var/log:logVol:2,/var/log/audit:auditVol:100%FREE}"
 AMIGENVGNAME="${SPEL_AMIGENVGNAME:-VolGroup00}"
-AMIUTILSSOURCE="${SPEL_AMIUTILSSOURCE:-https://github.com/ferricoxide/Lx-GetAMI-Utils.git}"
+AMIUTILSSOURCE="${SPEL_AMIUTILSSOURCE}"
```
[the remark this answers is not in the document]
Comment (lorengordon) [reply spel/scripts/amigen7-build.sh:21]: We'll eventually get around to forcing `nounset` on these scripts in spel and testing it with shellcheck, but right now it's not enforced. Note the next couple lines... Would prefer to deal with all of them at the same time.

**Review by kumoy — APPROVED** [review]

**Comment by lorengordon** [comment]
> With respect to the `max_retries`: is this likely to ever be required for other builder-defs (esp. Azure – maybe have @nortiz34 test)?

@ferricoxide I'm not sure on `max_retries` for azure builders... It's a feature of the aws sdk, and packer is just using the argument to override the value they otherwise pass to the sdk. It may not be implemented for azure at all.

**Pull request merged**